Matplotlib_common/read_UWLCM_arrays.py

```diff
@@ -7,7 +7,7 @@ def read_my_array(file_obj):
   line = line.split(" ")
   del line[0]
   del line[len(line)-1]
-  arr = map(float,line)
+  arr =list( map(float,line))
```

Comment:
what is the rationale to do this?

Reply:
https://stackoverflow.com/questions/50671360/map-in-python-3-vs-python-2

There was a change from Python2 to Python3, and list() is necessary to have the same output as map() in Python2